master/internal/task/allocation.go

```diff
@@ -395,6 +394,11 @@ func (a *Allocation) TaskContainerStateChanged(
 	switch msg.Container.State {
 	case cproto.Pulling:
 		a.state = model.MostProgressedAllocationState(a.state, model.AllocationStatePulling)
+		a.model.StartTime = ptrs.TimePtr(time.Now().UTC().Truncate(time.Millisecond))
+		if err := a.db.UpdateAllocationStartTime(a.model); err != nil {
```

Comment:
i think we can just log this error and move on. log an error that says something along the lines of 'this allocation maybe not be properly accounted for'.